apps/studio/tests/integration/helpers/db.ts

```diff
@@ -0,0 +1,11 @@
+import { db, sql } from "~server/db"
+
+import type { DB } from "~server/db"
+
+export async function resetTables(table: keyof DB, ...tables: (keyof DB)[]) {
+  for (const t of [table, ...tables]) {
+    await db.executeQuery(
+      sql`TRUNCATE TABLE ${sql.table(t)} CASCADE;`.compile(db),
+    )
+  }
+}
```

Comment:
this seems abit confusing to me - is this just to ensure that there's a minimum of 1 item?

Reply:
yup. also allows unlimited args using [variadic types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#variadic-tuple-types)